firebase-installations/src/androidTest/java/com/google/firebase/installations/FirebaseInstallationsInstrumentedTest.java

```diff
@@ -425,67 +437,54 @@ public void testGetAuthToken_multipleCalls_fetchedNewTokenOnce() throws Exceptio
         .that(task2.getResult().getToken())
         .isEqualTo(TEST_AUTH_TOKEN_2);
     verify(backendClientReturnsOk, times(1))
-        .generateAuthToken(anyString(), anyString(), anyString(), anyString());
+        .generateAuthToken(TEST_API_KEY, TEST_FID_1, TEST_PROJECT_ID, TEST_REFRESH_TOKEN);
   }
 
   @Test
   public void testGetAuthToken_multipleCallsForceRefresh_fetchedNewTokenTwice() throws Exception {
     when(mockPersistedFid.readPersistedFidEntryValue()).thenReturn(REGISTERED_FID_ENTRY);
-    // Use a custom ServiceClient to mock the network calls ensuring task1 is not completed
-    // before task2. Hence, we can test multiple calls to getAUthToken() and verify task2 waits for
-    // task1 to complete.
-    ServiceClient serviceClient = new ServiceClient();
+    // Use a mock ServiceClient for network calls with delay(1000ms) to ensure first task is not
+    // completed before the second task starts. Hence, we can test multiple calls to getAUthToken()
```

Comment:
nit: getAUthToken should be getAuthToken

Reply:
Done. Thanks.